Backpack/FlareView/Classes/BPKFlarePath.m

```diff
@@ -92,6 +92,8 @@ + (UIBezierPath *)flareViewPathForSize:(CGSize)size
         [self appendFlareToPath:path size:size flareHeight:flareHeight flarePosition:flarePosition];
     }
 
+    [path addLineToPoint:firstCornerCurveStart];
```

Comment:
This is needed to actually close the path. Because we only used it for masking before it worked even though it was left open

Reply:
Sounds like we should do close the path in the other use case too?